playbooks/install.py

```diff
@@ -14,7 +14,7 @@ def install_bench(args):
 	success = run_os_command({
 		'apt-get': [
 			'sudo apt-get update',
-			'sudo apt-get install -y git build-essential python-setuptools python-dev libffi-dev libssl-dev'
+			'sudo apt-get install -y git build-essential python-setuptools python-dev libffi-dev libssl-dev libldap2-dev libsasl2-dev'
```

Comment:
You should move these changes to ubuntu.yml as these are needed only for installation of frappe while all other required as dependencies to install Ansible, and run the script.